shared/src/main/kotlin/com/egm/stellio/shared/util/JsonLdUtils.kt

```diff
@@ -1,5 +1,7 @@
 package com.egm.stellio.shared.util
 
+import arrow.core.extensions.list.monoidal.identity
+import arrow.core.identity
```

Comment:
those 2 imports are not used

Reply:
yes, weird that detekt did not detect it